pkgs/applications/radio/sdr-j-fm/default.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, wrapQtAppsHook
+, pkg-config
+, qtbase
+, qwt
+, fftwFloat
+, libsamplerate
+, portaudio
+, libusb
+, libsndfile
+, airspy
+, librtlsdr
+, sdrplay
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sdr-j-fm";
+  version = "3.16";
+
+  src = fetchFromGitHub {
+    owner = "JvanKatwijk";
+    repo = "sdr-j-fm";
+    rev = "V${version}";
+    hash = "sha256-9B1ydbAqv2Qfuoljex0cS14PlOxS+bFQL5OiFDVYBAQ=";
+  };
+
+  cmakeFlags = [
+    # Upstream doesn't include the libairspy/ subdirectory in it's include
+    # paths
+    "-DAIRSPYLIB_INCLUDE_DIR=${lib.getDev airspy}/include/libairspy"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    qtbase
+    qwt
+    fftwFloat
+    libsamplerate
+    portaudio
+    libusb
+    libsndfile
+    airspy
+    librtlsdr
+    sdrplay
+  ];
+  postInstall = ''
+    # Weird default of upstream
+    mv $out/linux-bin $out/bin
+  '';
+
+  meta = with lib; {
+    description = "Sdr-j fm software";
```

```suggestion
    description = "SDR based FM radio receiver software";
```
(is a bit more descriptive and does not contain the name of the package)

---

> (is a bit more descriptive and does not contain the name of the package)

Thanks!